app/src/main/resources/templates/thymeleaf/fragments/base.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html th:fragment="base(content)" xmlns:th="http://www.thymeleaf.org" lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title th:text="${title} + ' &lt; Single Cell Expression Atlas &lt; EMBL-EBI'"></title>
+    <meta name="description" content="EMBL-EBI Single Cell Expression Atlas, an open public repository of single cell gene expression data">
+    <meta name="keywords" content="expression atlas, single cell expression, gene expression, baseline expression, functional genomics, public repository, repository, bioinformatics, europe, institute">
+    <meta name="author" content="EBI Functional Genomics Team – https://www.ebi.ac.uk/people/person/christina-ernst">
+    <meta name="HandheldFriendly" content="true" />
+    <meta name="MobileOptimized" content="width" />
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="theme-color" content="#70BDBD"> <!-- Android Chrome mobile browser tab color -->
+
+    <!-- Add information on the life cycle of this page -->
+    <meta name="ebi:owner" content="Christina Ernst <cernst@ebi.ac.uk>">
+
+    <!-- If you link to any other sites frequently, consider optimising performance with a DNS prefetch -->
+    <link rel="dns-prefetch" href="https://www.ebi.ac.uk/gxa/">
+    <link rel="dns-prefetch" href="https://www.ebi.ac.uk/arrayexpress/">
+    <link rel="dns-prefetch" href="https://www.ebi.ac.uk/">
```

Comment:
These are redundant as our website is already on www.ebi.ac.uk.
TYhey cause no harm but code clutter and potential extra work by the browser.
dns prefetch has only effect when used for a differnt domain than the website is hosted on, e.g. CDNs. So in this case, the emblstatic one could be.

```suggestion
    <link rel="dns-prefetch" href="https://ebi.emblstatic.net/">
```

Reply:
Ok, thanks. I have not noticed these, it make sense. I just replaced the JSP related code.